Description: Correct pod NAME issue in LedgerSMB/IS.pm
 Is missing a pod NAME entry altogether.
Bug: https://sourceforge.net/tracker/?func=detail&aid=3472606&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/IS.pm
+++ b/LedgerSMB/IS.pm
@@ -1,3 +1,11 @@
+=pod
+
+=head1 NAME
+
+LedgerSMB::IS - Inventory Invoicing
+
+=cut
+
 #====================================================================
 # LedgerSMB
 # Small Medium Business Accounting software
